.github/workflows/native-build.yml

```diff
@@ -14,6 +14,8 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
```

Comment:
commit msg: Is this fixing an actual issue / should the commit title prefix be `fix` ?

Reply:
IMO `fix` should only be used for user facing fixes in the application, CI changes should always use `ci` (and test changes always `test` and so on), because that makes the changelog more useful.

Reply:
I was also torn between the two, but as the fix *is* visible to the user when running `ort --version`, I decided to follow @fviernau's suggestion to make it a fix.

Reply:
Ok, I didn't actually look at the commits, I thought it was just a CI fix.